core/kazoo_ast/src/cb_api_endpoints.erl

```diff
@@ -465,7 +451,6 @@ path_name(<<_/binary>>=Module) ->
                ,[{'capture', 'all_but_first', 'binary'}]
                )
     of
-        {'match', [<<"about">>]} -> <<?CURRENT_VERSION/binary, "/about">>;
```

Comment:
If you look at cb_about, you will see there is nothing related to accounts going on in there. If `/v2/about` isn't working then `cb_about` should be fixed to return properly.